packages/wonder-blocks-core/util/add-style.md

```diff
@@ -0,0 +1,130 @@
+Adds a `style` property to components. A component can be either a custom React
+Component (e.g. Link, Button) or a JSX intrinsic (e.g. "div", "span").
+This helps you to manipulate and extend styles more easily by using the `styles`
+prop as an array of different style objects.
+
```

Comment:
Since we don't have a proper types inside Styleguidist it might be worth writing up exactly what the `style` property can take (for example `style={object|false|null|undefined|array<object|...>}`. I dunno, something that makes it clear what all the types of valid inputs are?

Reply:
Thanks for the suggestion!. I made some changes on that part and borrowed some ideas from @somewhatabstract's Timing documentation :)
https://deploy-preview-463--wonder-blocks.netlify.com/#!/AddStyle